Resources/Prototypes/Roles/MindRoles/mind_roles.yml

```diff
@@ -158,6 +230,15 @@
   - type: MindRole
     antagPrototype: TraitorSleeper
 
+- type: entity
+  parent: MindRoleTraitor
+  id: MindRoleTraitorReinforcement
```

Comment:
Good find, there is a funny story here.
As the name suggests, this was meant to be used for Traitor Reinforcements, set on the TraitorReinforcement gamerule. But they by some oversight or accident I apparently configured them to use regular MindRoleTraitor instead. This means their starting mind role is Solo Antagonist. How was this not spotted? Because all ghostroles can now also specify a roletype on the GhostRoleComponent. And all the traitor reinforcements also get a _second_ starting mindrole from GhostRoleComponent, which is correctly set to Team Antagonist, and since that one comes later, it overrides the first role type. The joys of various complex systems working on top of each other :godo:
I have now correctly set the reinforcements to get traitor as their original role, too, but I'm looking into whether there is a clean way to start out without redundant mind roles. It should not cause any issues that they have two with the same role type, though.